### DISCUSSION TITLE
Square root power does not work

### DISCUSSION THREAD

**Discussion by Dani-l4** [KaTeX/KaTeX#3925] [Q&A]
Hello
I got a problem where square root power not working
It goes like:
[[\sqrt{8 \over 9}]] -> *fine looking sqrt*
[[\sqrt[3]{8 \over 9}]] -> [[\sqrt[3]{8 \over 9}]]

If I remove the sqrt power its woking fine
I'm using 0.10.0 via https://cdn.jsdelivr.net/npm/katex@0.10.0-rc.1/dist/katex.js
Tried to switch to a fresh version (current is 0.16.9) via https://cdn.jsdelivr.net/npm/katex@0.16.9/katex.min.js
unsuccessfully (actually all formulas stopped working altogether)

Help me please

**Comment by edemaine** [comment]
Maybe try `[[{\sqrt[3]{8 \over 9}}]]` i.e. wrapping in braces?

It sounds like your issue is with whatever calls KaTeX, not KaTeX itself. What is the `[[...]]` notation?

**Reply by Dani-l4** [reply]
Yes, you was right, that was problem from my side. Sorry, thanks. You actually help

**Reply by andmartynov** [reply]
Hello i am the next one who had a problem with this software. This notation is used inside some of our projects to create html based on text format. the problem was that the regex didn't recognize single square brackets inside of double as part of the formula for the KaTeX, so I replaced this:
```answer = re.sub(r"\[\[([^\[\]]+)\]\]", lambda m:f'<span class="ktx">{m.group(1)}</span>', answer)```

with this:
```answer = re.sub(r"\[\[((?:[^\[\]]+|\[[^\[\]]*\])*)\]\]", lambda m:f'<span class="ktx">{m.group(1)}</span>', answer)```

And now it works. Thank you for help! New regex was made with chatgpt help.
